File: backend/Models/Entities/Charger.cs
namespace backend.Models.Entities
{
    public class Charger
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public DateTime CreatedAt { get; set; }
        public int CreatedById { get; set; }
        public virtual User CreatedBy { get; set; }
        public DateTime LastSyncAt { get; set; }
        public bool Active { get; set; }

        public virtual ICollection<Event> Events { get; set; }
    }
}

File: android/app/src/main/java/hr/foi/air/wattsup/network/models/Charger.kt
package hr.foi.air.wattsup.network.models

import java.util.Date

data class Charger(
    val id: Int,
    val name: String,
    val latitude: Double,
    val longitude: Double,
    val createdAt: Date,
    val createdBy: Int,
    val lastSyncAt: Date,
    val active: Boolean
)